### DISCUSSION TITLE
Email confirmation

### DISCUSSION THREAD

**Discussion by leadsmkt** [JulianPrieber/littlelink-custom#601] [General]
When a user registers, an email is sent to them to validate their email address. Could you tell me in which file I can modify the content of the message?
Thank you

**Comment by JulianPrieber** [comment]
Easiest way is to modify the contents found in resources/lang/xy/messages.php

**Reply by leadsmkt** [reply]
Thank you for you reply.
When i check the file 'message.php" (en or in others languages), i can see  : 

-  # Signup notification email  (line 100).  (for admin)
- # Email verification email (message displayed on the page after the user's registration)

But, i can't see the content for the mail sent to the user 

**H_ello!
Please click the button below to verify your email address.
If you did not create an account, no further action is required.

Regards**_

Please let me know if you have an idea?

**Reply by JulianPrieber** [reply]
https://github.com/LinkStackOrg/LinkStack/blob/main/resources/lang/en/messages.php#L113-L116

**Reply by JulianPrieber** [reply]
Did we not translate that part?

**Reply by JulianPrieber** [reply]
Maybe in here I'm not sure
https://github.com/LinkStackOrg/LinkStack/tree/main/resources/views/auth

**Reply by JulianPrieber** [reply]
The text comes from upstream here: https://github.com/khzg/littlelink-admin

**Reply by JulianPrieber** [reply]
Default Laravel behavior
https://laracasts.com/discuss/channels/laravel/change-email-verification-body

**Reply by leadsmkt** [reply]
Thank you again, I'll try to do something but it might be above my capabilities! We will see ...
